AutonomousMobileRobot/ReturnMessageROS/examples/example-normalized.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
same issue as previous payload with string and object, anyhow are valid payloads